.github/workflows/helm-release-chart.yml

```diff
@@ -0,0 +1,38 @@
+name: Release Helm Chart
+on:
+  push:
```

Comment:
could we add back a workflow dispatch? i could instantly dispatch a run for `v4.7.0`.

Reply:
No problem!